tests/integration/override_targets/Loader.java

```diff
@@ -0,0 +1,14 @@
+package com.google.ar.sceneform.assets;
+
+/**
+ * Some class in com.google.ar.sceneform:rendering:aar:1.10.0 loads
+ * this class, but there is no class Loader in
+ * com.google.ar.sceneform:assets:1.10.0 or any other aar in sceneform.
+ * This is only to satisfy the ImportDepsChecker check since none of
```

Comment:
`..satisfy aar_import's built-in ImportDepsChecker check..`

Reply:
Clarified